tests/gem5/gpu/README.md

```diff
@@ -1,7 +1,12 @@
 # GPU
 
-These tests do random checks to the Ruby GPU protocol within gem5. Additionally, these tests run SE mode GPU tests.
-To run these tests by themselves, you can run the following command in the **gem5** directory:
+These tests:
+
+1. Carry out randomly generated checks to the Ruby GPU protocol within gem5.
+2. Run a set SE mode GPU simulations.
+
+To run the SE mode by themselves, you must run within a `gcn-gpu` Docker
```

Comment:
I'm assuming this is still required outside testlib?  Because @BobbyRBruce has a commit in this chain that seems to remove the docker for this test ...